test/jdk/java/util/Currency/CurrencyTest.java

```diff
@@ -238,8 +238,9 @@ private static Stream<Arguments> expectedFractionsProvider() {
                 Arguments.of("BHD", 3), Arguments.of("IQD", 3),
                 Arguments.of("JOD", 3), Arguments.of("KWD", 3),
                 Arguments.of("LYD", 3), Arguments.of("OMR", 3),
-                Arguments.of("TND", 3), Arguments.of("TRL", 0), // Turkish Lira
-                Arguments.of("TRY", 2)
+                // Old and New Turkish Lira
+                Arguments.of("TRL", 0), Arguments.of("TRY", 2),
+                Arguments.of("TND", 3)
```

Comment:
Probably `TND` moving upward before the comment would be clearer, like:
```
                Arguments.of("LYD", 3), Arguments.of("OMR", 3),
                Arguments.of("TND", 3),

                // Old and New Turkish Lira
                Arguments.of("TRL", 0), Arguments.of("TRY", 2)
```

Reply:
Adjusted `TND` upwards